style.css

```diff
@@ -14,4 +14,45 @@ h1 {
 .flex-container1 {
   display: flex;
   background-color: lightsalmon;
+  flex-direction: column;
+}
+
+.sunflower-answer {
+  background-color: chocolate;
+  border: 0cqmax;
```

Comment:
I think this value ist not valid. You could define the style, the width and the color of the border here